src/sentry/monitors/consumers/check_in.py

```diff
@@ -25,9 +25,7 @@
 logger = logging.getLogger(__name__)
 
 
-def process_message(message: Message[KafkaPayload]) -> None:
-    wrapper = msgpack.unpackb(message.payload.value)
-
+def _process_message(wrapper: Dict) -> None:
     params = json.loads(wrapper["payload"])
```

Comment:
We should probably add a schema here at some point.

Reply:
ah. yeah.